src/Hazelcast.Net.Win32/Hazelcast.Net.Win32.csproj

```diff
@@ -45,6 +45,7 @@
   
   <ItemGroup>
     <PackageReference Include="System.Memory" Version="4.5.4" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
```

Comment:
Don't we also need additional configuration as described for  [SourceLink](https://github.com/dotnet/sourcelink#using-source-link-in-net-projects)? Or this is only the first step?

Reply:
I _think_ all the additional configuration was already in the file and just that line was missing. Can you have a look at the rest of the file?